NLog.Elmah/ElmahTarget.cs

```diff
@@ -33,6 +33,16 @@ public sealed class ElmahTarget : TargetWithLayout
         /// </summary>
 		public bool LogLevelAsType { get; set; }
 
+        /// <summary>
+        /// Use <see cref="LogEventInfo.LoggerName"/> as source if <see cref="LogEventInfo.Exception"/> is <c>null</c>.
+        /// </summary>
+        public bool LoggerNameAsSource { get; set; }
```

Comment:
Why not make it a Layout that is initially assigned to `${logger}` ?

Reply:
Smart one!

Reply:
Good idea :)